test/views/ConfirmationViewSpec.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.ViewBaseSpec
+import messages.ConfirmationMessages
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.twirl.api.Html
+import views.html.ConfirmationView
+
+class ConfirmationViewSpec extends ViewBaseSpec {
+
+  private lazy val injectedView: ConfirmationView = app.injector.instanceOf[ConfirmationView]
+
+  "Rendering the Confirmation page" when {
+    "no errors exist" should {
```

Comment:
need a test for when a reference is passed into the view it does render how we expect it